scripts/run-unit-tests-in-docker

```diff
@@ -6,7 +6,7 @@ ROUTING_RELEASE_DIR="${BASEDIR}/.."
 # COMPONENT takes only single package name
 COMPONENT=$1
 PACKAGE=$2
-IMAGE="cloudfoundry/cf-routing-pipeline"
+IMAGE="test-1.19"
```

Comment:
Was this included accidentally? I couldn't run the tests on my machine and had to change it back to cloudfoundry/cf-routing-pipeline.